### ISSUE TITLE
TODO: Add check of n_cores

### ISSUE THREAD

**Issue by juraskov** [duartegroup/mlp-train#94]
Variable mlt.Config.n_cores = 35 is not checked against the number of n_configs_iter before the beginning of the training. We should implement checks and warnings or adapt the number of cores automatically. Otherwise, the code crashes only after completing the first round of training, which can waste significant time for large systems.

**Comment by lucasaraujodelimadias** [comment]
Up.

I noticed that the parallelization method divides the number of cores by the number of structures. So, while generating an initial dataset by converting .xyz to .npz using ORCA, if I provide more structures than processors, the jobs take forever because only one core is used to compute energies and forces. I think it would be nice to have something like mlt.Config.orca_cores that always uses a fixed number of cores for the ORCA calculations.
